src/main/java/com/sprint/mission/discodeit/entity/BaseUpdatableEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.sprint.mission.discodeit.entity;
+
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import org.springframework.data.annotation.LastModifiedDate;
+
+import java.time.Instant;
+
+@Getter
+@MappedSuperclass
+@NoArgsConstructor
+public abstract class BaseUpdatableEntity extends BaseEntity {
```

Comment:
`@EntityListeners(AuditingEntityListener.class)` 이 애노테이션 없으면 `@LastModifiedDate` 동작 안하는걸로 알고 있는데요, 지훈님 로컬환경에서 테스트했을 땐 정상동작하나요?
```suggestion
@EntityListeners(AuditingEntityListener.class)
public abstract class BaseUpdatableEntity extends BaseEntity {
```